src/vs/workbench/api/common/extHostSCM.ts

```diff
@@ -887,6 +910,8 @@ class ExtHostSourceControl implements vscode.SourceControl {
 		this._acceptInputDisposables.dispose();
 		this._actionButtonDisposables.dispose();
 		this._statusBarDisposables.dispose();
+		this._historyProviderDisposable.dispose();
+		this._artifactProviderDisposable.dispose();
```

Comment:
nit: `MutableDisposable`s registered on class init are a nicer way of handling this. That way the variable declaration and registering are in a single spot.